src/coreComponents/constitutive/solid/LinearElasticTransverseIsotropic.hpp

```diff
@@ -375,17 +389,16 @@ class LinearElasticTransverseIsotropic : public SolidBase
    */
   void setDefaultPoissonsRatioTransverse( real64 const input )
   {
-    m_defaultPoissonTransverse = input;
+    m_defaultPoissonsRatioTransverse = input;
   }
 
-
   /**
    * @brief Getter for default axial Poisson's ratio
    * @return The value of the default axial/transverse Poisson's modulus.
    */
   real64 getDefaultPoissonsRatioAxialTransverse()
```

Comment:
This should be `const`.

Reply:
const is now added to all the get... functions in this class